core-tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/connector/Dummy.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2018-2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.reactive.messaging.tck.connector;
+
+import org.eclipse.microprofile.reactive.messaging.MessagingProvider;
+
+public class Dummy implements MessagingProvider {
```

Comment:
I think if we are going to keep it? we need to explain why it exists (to me too :-) )